packages/frontend/src/pages/PasswordReset.tsx

```diff
@@ -1,118 +1,133 @@
-                                            type="submit"
-                                            intent={Intent.PRIMARY}
-                                            text="Save"
-                                            loading={resetMutation.isLoading}
-                                        />
+                                        <Stack spacing="lg">
+                                            <PasswordInput
+                                                label="Password"
+                                                name="password"
+                                                placeholder="Enter a new password"
+                                                disabled={
+                                                    passwordResetMutation.isLoading
+                                                }
+                                                required
+                                                {...form.getInputProps(
+                                                    'password',
+                                                )}
+                                            />
 
-                                        <Center mt="lg">
-                                            <Anchor
-                                                component={Link}
-                                                to="/login"
+                                            <Button
+                                                type="submit"
+                                                loading={
+                                                    passwordResetMutation.isLoading
+                                                }
                                             >
-                                                Cancel
-                                            </Anchor>
-                                        </Center>
-                                    </Form>
+                                                Save
+                                            </Button>
+
+                                            <Center>
+                                                <Anchor
+                                                    component={Link}
+                                                    to="/login"
+                                                >
+                                                    Cancel
+                                                </Anchor>
+                                            </Center>
+                                        </Stack>
+                                    </form>
                                 </>
                             ) : (
                                 <>
-                                    <Title>Success! ✅</Title>
-                                    <Subtitle>
+                                    <Title order={3} ta="center" mb="md">
+                                        Success!
```

Comment:
I think we can keep the emoji here

Reply:
@PriPatel pointed out in another area that we should remove the 🙈  emoji, would you say we keep it or remove it here? 
was just following what I thought would be the final product: plain canvas+no glitter

Reply:
ok, ignore this! 👍🏼